independent-publisher-connectors/ShipStation/apiProperties.json

```diff
@@ -0,0 +1,12 @@
+{
+  "$schema": "../../schemas/paconn-apiProperties.schema.json",

```

Comment:
@SmolSoftBoi Can you please let me know the reason for adding this property? This is neither allowed nor valid, Can you please make sure that you are validating your connector artifacts with paconn tool.

Reply:
Hi @SmolSoftBoi, I am following up to check on how updating your artifacts are going.
Please let me know if you have any questions.
Thank you very much